samples/unit-tests/loading/hideduration/demo.js

```diff
@@ -0,0 +1,100 @@
+
+QUnit.test('Show and hide duration', function (assert) {
```

Comment:
This test fails unreliably in the browser (using utils), due to float errors. We should use `QUnit.close` or `toPrecision` on all asserts.

Reply:
I will restore the original timing, that has no problem with the float precision.